tests/test_missing_treatments.py

```diff
@@ -0,0 +1,50 @@
+from doc_utils import DocUtils
+from doc_utils.errors import InvalidTreatment, MissingFieldError
+
+def test_return_empty_string(sample_document):
```

Comment:
@pytest.mark.parametrize should be useful here as there's a lot of repeated code.